microprofile.jdt/com.redhat.microprofile.jdt.core/schema/projectType.exsd

```diff
@@ -0,0 +1,111 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Schema file written by PDE -->
+<schema targetNamespace="com.redhat.microprofile.jdt.core" xmlns="http://www.w3.org/2001/XMLSchema">
+<annotation>
+      <appinfo>
+         <meta.schema plugin="com.redhat.microprofile.jdt.core" id="projectType" name="%projectType.name"/>
```

Comment:
When you use %projectType.name, it search the label in plugin.properties. Declare it like https://github.com/redhat-developer/quarkus-ls/blob/7c555b9351c1ea743c4368702dd9864695171cfa/microprofile.jdt/com.redhat.microprofile.jdt.core/plugin.properties#L13